config/locales/idv/en.yml

```diff
@@ -82,6 +82,10 @@ en:
         heading: We could not find records matching your personal information.
         warning: Please check the information you entered and try again. Common mistakes
           are an incorrect Social Security number or ZIP Code.
+      setup:
+        fail_html: 'Please <a href="%{contact_form}">fill out our contact form</a> and
+          enter the code <b>LG99</b> in the detail field.'
```

Comment:
let's turn this code into a config value and put a fake one in the config. I would like to make sure people don't read our source code to find a way to escalate/shortcut support